### PR TITLE
create new issue template for requesting a rule

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- add a new issue template

- fix a typo in generic issute template

#### Rationale:

There appeared a list of questions which are often asked when a new SCAP rule is proposed. So why not to put them to test here upstream?

### CODE REVIEW

File: .github/issue_template/new_rule.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+```
+kernelopts=arg1 arg2 audit=1 arg3
+```
+
+## How to remediate
+
+example: Ensure that the argument is present in the kernelopts=... line.
+
+###Does any command need to be run?
```
Comment (ggbecker) [inline .github/issue_template/new_rule.md:55]:
```suggestion
### Does any command need to be run?
```

add an space between `#` and `Does`

**Review by ggbecker — CHANGES_REQUESTED** [review]
I think examples can be turned into comments... so people won't need to actually delete them before creating the issue.

Comments are represented by `<!-- commented text --!>`

<!-- commented text --!>

**Comment by vojtapolasek** [comment]
Good suggestion, fixed.

File: .github/issue_template/new_rule.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+<!---
+example:
+
+```
+sudo grep 'kernelopts.*audit=1.*' /boot/grub2/grubenv
+```
+-->
+
+### Is it order dependent? (does it need to be at certain place in the file?
```
Comment (ggbecker) [inline .github/issue_template/new_rule.md:42]:
```suggestion
### Is it order dependent? (does it need to be at certain place in the file?)
```

**Review by ggbecker — APPROVED** [review]
LGTM. Failing tests have no bearing to these changes.

**Pull request merged**